tests/native_tests/language.hy

```diff
@@ -12,6 +12,16 @@
   (assert (isinstance sys.argv list)))
 
 
+(defn test-hex []
+  "NATIVE: test hex"
+  (assert (= 0x80 128)))
+
+
+(defn test-octel []
```

Comment:
(The standard spelling [is](https://en.wikipedia.org/wiki/Octal) *octal*, with an *a*.)